library/lua/dfhack/workshops.lua

```diff
@@ -126,19 +126,6 @@ jobs_workshop={
             job_fields={job_type=df.job_type.CatchLiveFish}
         }, -- no items?
     },
-    [df.workshop_type.Still]={
```

Comment:
Nope this file (workshops.lua) is out of date. Though not much depends on it but there are some jobs that are no longer hardcoded.